README.md

```diff
@@ -146,3 +146,18 @@ over the set of tested equations and some specific examples may differ.
 ### Stiff DDEs
 
 - The Rosenbrock methods, specifically `Rodas5`, perform well.
+
+### Parameter Estimation
+
+- Broadly two different approaches have been used, Bayesian Inference and Optimisation 
+  algorithms.
+- In general it seems that the optimisation algorithms perform more accurately but that can be 
+  attributed to the larger number of data points being used in the optimisation cases, Bayesian 
+  approach tends to be slower of the two and hence lesser data points are used, accuracy can 
+  increase if proper data is used.
+- Within the different available optimisation algorithms, BBO, GN_CRS2_LM, LD_SLSQP, LN_BOBYQA 
+  and LN_NELDERMEAD perform the best.
+- The different backends options available for Bayesian method offer some tradeoffs beteween
+  time, accuracy and control. It is observed that sufficiently high accuracy can be observed with 
```

Comment:
Do we have enough information to suggest heuristics yet?

Reply:
That is true, but at face value one can say something about the general trend though?

Reply:
Oh no, I meant do we have enough information to suggest an in-category heuristic trend? Which of the 3 are generally faster, or more accurate?

Reply:
For faster we have the time from the `@time` macro and for accuracy I am just using an euclidean of sorts in my head I would say. I get that it isn't a concrete statement this way, would you prefer no mention of specific algorithms?

Reply:
I would prefer a mention of what's fast and what's accurate if you think they are already starting to differentiate from each other.